Makefile

```diff
@@ -1,13 +1,24 @@
-RHEL5=$(shell grep -q -i 'release 5' /etc/redhat-release)
+RHEL5=$(shell grep -i 'release 5' /etc/redhat-release)
+RHEL6=$(shell grep -i 'release 6' /etc/redhat-release)
 PYTHON3=$(shell command -v python3 && command -v pip3)
 USE_SELINUX=$(shell test -d /sys/fs/selinux && echo ":Z")
 DOCKERFILE  ?= $(CURDIR)/images/Dockerfile.el7
+TEST_INSTALL_CMD=$(PIP) install -r test-requirements.txt --user
+
 ifneq ($(PYTHON3),)
 PYTHON ?= python3
 PIP ?= pip3
 else
 PYTHON ?= python
 PIP ?= pip
+endif 
+
+ifneq ($(RHEL5),)
+TEST_INSTALL_CMD=
+endif
+
+ifneq ($(RHEL6),) # python 2.6 busted for flake8
+TEST_INSTALL_CMD=$(PIP) install -r test-requirements-py26.txt --user
```

Comment:
you don't need a special `requirements.txt` for EL6, you can just write:

```
flake8; python_version >= '2.7'
mock
unittest2
```

[pip docs](https://pip.pypa.io/en/stable/reference/pip_install/#requirement-specifiers)
[pep 508](https://www.python.org/dev/peps/pep-0508/#environment-markers)

Reply:
Wow nice, updated!